.travis.yml

```diff
@@ -0,0 +1,50 @@
+language: node_js
+node_js:
+  - 12
+before_install:
+  # Before_install section
+  - ignore
+before_script:
+  # Before_script section
+  - ignore
+
+env:
+  global:
+    - SOURCE_TEST_SITE_BRANCH=travis-setup
+    - TARGET_JDI_LIGHT_BRANCH=gh-pages
+
+stages:
+  - prepare
+  - compile
+  - test
+  - merge
+
+jobs:
```

Comment:
we need jobs only when time to run all tasks is more than 50 minutes, in other cases you should use only one job

Reply:
I've removed the unnecessary placeholder jobs and left only two: test and merge.
Once we properly set up the 'test' job, it would be used as quality gate to decide whether merge should be triggered.